components/WhatICanHelpYouWith/organisms/WhatICanHelpYouWith.tsx

```diff
@@ -13,26 +13,32 @@ import {
   LowerContainer,
   MobileCTAContainer,
 } from 'components/WhatICanHelpYouWith/organisms/WhatICanHelpYouWith.styles';
+import PsychoLink from 'components/atoms/PsychoLink';
+import Footer from 'components/Footer/Footer';
 
 const WhatICanHelpYouWith = () => (
-  <BackgroundWrapper color={Blue010}>
-    <Container>
-      <Top />
-      <LowerContainer>
-        {items.map(
-          (item, index) => <Unit item={item} key={index} />,
-        )}
-      </LowerContainer>
-      <MobileCTAContainer>
-        <PrimaryCTAButton
-          text="Árlista megtekintése"
-          color={Grey010}
-          backgroundColor={Green050}
-          linkHref=""
-        />
-      </MobileCTAContainer>
-    </Container>
-  </BackgroundWrapper>
+  <>
```

Comment:
Instead of adding the Footer to every page separately, you could consider adding it to the _app.tsx, and deciding there which routes should it display or not.

Reply:
I tend to agree with this, probably if we only need to skip footer on a single page, a blacklist is better than a whitelist.